msal/application.py

```diff
@@ -284,7 +284,7 @@ def get_authorization_request_url(
             Can be one of "consumers" or "organizations" or your tenant domain "contoso.com".
             If included, it will skip the email-based discovery process that user goes
             through on the sign-in page, leading to a slightly more streamlined user experience.
-            More information on possible values 
```

Comment:
Is this change needed ?

Reply:
Absolutely not. :-) And really glad that you notice this!

It was, however, an unnecessary trailing white space in the first place, which we failed to notice it when [it was introduced in this PR](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/197/files#diff-f7ae931bcdab4b14f3a256baf7d6922aR287). :-(

That has to be fixed otherwise our future PR(s) would still encounter this conversation. I/we could have create a dedicate PR only to fix that trailing space.

By the way, do you happen to know whether PyCharm can be configured to automatically trim all the trailing spaces? That's the first thing I do whenever I install a new editor.

Reply:
Yes unfortunate that it got introduced in the first place, I guess there should be a setting to configure this in PyCharm,  will do this configuration so these kind of spaces aren't introduced next time onwards :)